src/index.html

```diff
@@ -17,6 +17,43 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="nav">
+        <div class="logo">
+          <img
+            src="images/logo.png"
+            alt="logo Moyo"
+          />
+        </div>
+        <nav>
+          <ul class="link">
+            <li class="box box-1">
+              <a
+                href="index.html"
+                class="is-active"
+              >
+                Apple
+              </a>
+            </li>
+            <li class="box box-2"><a href="index.html">Samsung</a></li>
```

Comment:
Youn need to fix code style and class everywhere
```suggestion
            <li class="nav__item">
              <a class="nav__link" href="index.html">Samsung</a>
            </li>
```